### PR TITLE
CYB-439, apply Darlene's work to fix: deploy.sh not writing sidekiq pid

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CYB-439

Per Darlene's attachment to Jira above.

### CODE REVIEW

File: bin/deploy.sh [hunk 1]
```diff
@@ -26,14 +26,12 @@ fi
...
-if [ ${active_pid} ]; then
-  pids_dir="${appHome}/tmp/pid"
-  mkdir -p "${pids_dir}"
-  sidekiq_pid="${pids_dir}/sidekiq.pid"
-  echo "${active_pid}" > "${sidekiq_pid}"
-  sidekiqctl stop "${sidekiq_pid}" \
+# Stop Sidekiq 
+sidekiq_pid_file="${appHome}/tmp/pids/sidekiq.pid"
+sidekiq_pid=$(/bin/cat $sidekiq_pid_file)
+active_pid=$(pgrep -U app_calcentral -f "sidekiq [0-9.]* datacultures \[[0-9]* of [0-9]* busy\]")
```
Comment (nicolaasmatthijs) [inline bin/deploy.sh]: This should be using the `app_user` variable

**Comment by nicolaasmatthijs** [comment]
Left some minor review. Back to @johncrossman for follow-up

**Comment by johncrossman** [comment]
Thanks @nicolaasmatthijs   Fixed.

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged.

**Pull request merged**